src/components/macros/playSound.js

```diff
@@ -97,6 +97,14 @@ function PlaySound({updateArgs, args, stations, clients, playSound}) {
           value={sound.channel}
           onChange={e => updateSound("channel", e.target.value.split(","))}
```

Comment:
This input should be disabled/hidden if `preserveChannels` is true, since it won't have any effect. I would actually move the `preserveChannels` checkbox above this one to have users select it before editing the channel input.